commands/setup.go

```diff
@@ -62,6 +63,10 @@ func Setup() {
 	}
 
 	if !cfg.Setup.SkipRegistration {
+		if err := config.IsWritable(configFilepath); err != nil {
```

Comment:
we can extract it from this `if`. command may fail even with `--skip-registation` flag and it's better to have the same message in both cases.